polaris-react/src/components/IndexTable/IndexTable.stories.tsx

```diff
@@ -1210,6 +1218,7 @@ export function WithBulkActionsAndSelectionAcrossPages() {
     >
       <LegacyCard>
         <IndexTable
+          condensed={useBreakpoints().smDown}
```

Comment:
WithBulkActionsAndSelectionAcrossPages story doesn't have proper paddings on Indextable when condensed
<img width="1332" alt="image" src="https://github.com/Shopify/polaris/assets/104942025/e59d0c41-c8d7-474d-bb85-4d1a3c1e8c34">

Reply:
Removed this funky padding, not sure why that was there 👍